src/test/java/org/nibor/autolink/LinkExtractorIterableTest.java

```diff
@@ -1,11 +1,15 @@
 package org.nibor.autolink;
 
-import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
 
 import java.util.Iterator;
+import java.util.List;
 import java.util.NoSuchElementException;
 
-import static org.junit.Assert.*;
+import org.junit.Test;
 
 public class LinkExtractorIterableTest {
```

Comment:
Until now, this test was only about the iterable part. Could you rename it to `LinkExtractorTest`?